docs/how-to/artifact.rst

```diff
@@ -60,15 +60,31 @@ Below, we have included a list of currently supported artifact handlers and thei
       - json
       - Artifacts written using :py:meth:`json.dump` and read using :py:meth:`json.load`
       - N/A
-    * - :py:class:`lazyscribe.artifacts.joblib.JoblibArtifact`
-      - joblib
-      - Artifacts written using :py:meth:`joblib.dump` and read using :py:meth:`joblib.load`
-      - ``joblib``
     * - :py:class:`lazyscribe.artifacts.yaml.YAMLArtifact`
       - yaml
       - Artifacts written using :py:meth:`yaml.dump` and read using :py:meth:`yaml.load`. You can specify the dumper using the ``Dumper`` keyword argument and the loader using the ``Loader`` keyword argument. Defaults to :py:class:`yaml.FullDumper` and :py:class:`yaml.SafeLoader` respectively if not specified.
       - ``PyYAML``
 
+We also provide first-party supported artifact handlers through separately distributed projects:
+
+.. list-table:: Builtin artifact handlers
+    :header-rows: 1
+
+    * - Alias
+      - Description
+      - Installation
+    * - joblib
+      - Artifacts written using :py:meth:`joblib.dump` and read using :py:meth:`joblib.load`
+      - `lazyscribe-joblib <https://github.com/lazyscribe/lazyscribe-joblib>`_
+    * - csv
+      - Artifacts written to CSV files using PyArrow
+      - `lazyscribe-arrow <https://github.com/lazyscribe/lazyscribe-arrow>`_
+    * - parquet
+      - Artifacts written to parquet files using PyArrow
+      - `lazyscribe-arrow <https://github.com/lazyscribe/lazyscribe-arrow>`_
```

Comment:
```suggestion
    * - parquet
      - Artifacts written to parquet files using Parquet
      - `lazyscribe-parquet <https://github.com/lazyscribe/lazyscribe-parquet>`_
```

Reply:
We don't have a `lazyscribe-parquet` package. Both the `csv` and `parquet` handlers are made available via `lazyscribe-arrow`.